### ISSUE TITLE
GDB does not work

### ISSUE THREAD

**Issue by ghanan94** [ish-app/ish#47]
I tried to install gdb with the hope of debugging why tmux wouldn’t work (another issue I submitted) and found out that gdb doesn’t work either :(

\# apk add gdb
...
\# gdb tmux
Segmentation Fault
\# gdb -h
Segmentation Fault
\# gdb
Segmentation Fault
\#

**Comment by tbodt** [comment]
I've never tried to run gdb so I'm not surprised this happens. In any case, gdb relies on ptrace which is totally unimplemented.

**Comment by Mnpn03** [comment]
```
Device:~# apk add gdb
OK: 92 MiB in 45 packages
Device:~# gdb
Segmentation fault
Device:~# dmesg
FIXME socketcall 16
13 page fault on 0x50 at 0x567336fd
```

**Comment by ntindle** [comment]
`34 missing syscall 136` When debugging hello world. 
```c
#include <stdio.h>
int main()
{
   printf("Hello, World!");
   return 0;
}
```

If saved as `prog1.c`

```bash
gcc  -Wall -Werror -ansi -pedantic-error
s -g prog1.c -o prog1.x
gdb prog1.x
```

```
Reading symbols from prog1.x...
(gdb) run

**Comment by saagarjha** [comment]
Should be fixed by #736, so closing this for now. Much of ptrace is still unimplemented, but you should be able to debug simple single-threaded programs now.
